packages/medusa/src/models/product-category.ts

```diff
@@ -94,9 +94,8 @@ export class ProductCategory extends SoftDeletableEntity {
  *       type: object
  *       description: A product category object.
  *   parent_category:
- *     description: Available if the relation `parent_category` is expanded.
+ *     description: A product category object. Available if the relation `parent_category` is expanded.
```

Comment:
Question: is it not returned by default? We have to pass it to the expand query params to get it?

Reply:
Thanks for fixing this! Its not eager loaded by default, but this is for all potential options for the model right?

Reply:
I am asking myself if the expand notion should even appear in the model description. It is an api query param. Wdyt?